docs/data-sources/edgegateway_ip_set

```diff
@@ -0,0 +1,40 @@
+---
+page_title: "cloudavenue_edgegateway_ip_set Data Source - cloudavenue"
+subcategory: "Edge Gateway (Tier-1)"
+description: |-
+  The cloudavenue_edgegateway_ip_set data source allows you to retrieve information about an IP Set rule on an Edge Gateway.
+---
+
+# cloudavenue_edgegateway_ip_set (Data Source)
+
+The `cloudavenue_edgegateway_ip_set` data source allows you to retrieve information about an IP Set rule on an Edge Gateway.
+
+## Example Usage
+
+```terraform
+data "cloudavenue_edgegateway_ip_set" "example" {
+  name            = "example"
+  edge_gateway_id = cloudavenue_edgegateway.example[0].id
```

Comment:
A confusion beetwen list of edgegateways and an edgegateway
edge_gateway_id = data.cloudavenue_edgegateways.example[0].id